### ISSUE TITLE
Double error TypeError: null is not an object

### ISSUE THREAD

**Issue by oleksandrbazil** [350d/stylish#29]
TypeError: null is not an object (evaluating 'settings.minify')
(anonymous function) — script.js:54

Safari Version 10.1 (12603.1.30.0.34)

**Comment by 350d** [comment]
Hi! Can you provide more details about extension version, page url and style id?

**Comment by michaeldhopkins** [comment]
Hi, I came here looking for this issue. I am experiencing this in Stylish 2.0.0 in Safari 10.1.1. The error occurs once per load for each matching extension. For example, I have two reddit.com extensions, so I see the error twice when I load https://www.reddit.com/r/AskReddit/ . I've tested this with a directory theme (Reddit No Frills Dark+ https://userstyles.org/styles/133277) and a custom style.

The triggering line is :56 		if (settings.minify == 'on') css = minify_css(css);
